hmda/src/main/scala/hmda/validation/rules/lar/validity/V629_1.scala

```diff
@@ -0,0 +1,27 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.model.filing.lar.enums.{
+  EthnicityObservedNotApplicable,
+  InvalidEthnicityObservedCode,
+  NotVisualOrSurnameEthnicity,
+  VisualOrSurnameEthnicity
+}
+import hmda.validation.dsl.ValidationResult
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+object V629_1 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "V629-1"
+
+  override def parent: String = "V629"
+
+  override def apply(input: LoanApplicationRegister): ValidationResult = {
```

Comment:
Just to keep it consistent with other edits, we're doing `apply(lar: LoanApplicationRegister)` (as opposed to "input")